assets/js/theme.js

```diff
@@ -66,11 +66,12 @@ const hideSearch = () => {
 // CTRL + K -> Open search
 // ESC -> Hide search
 document.addEventListener("keydown", function (e) {
-  if (e.keyCode === 75 && (e.ctrlKey || e.metaKey)) {
+  if (e.key === 75 && (e.ctrlKey || e.metaKey)) {
```

Comment:
the object `key` stores [string values](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values) for keys. `keyCode` was correct, now `Ctrl` / `Command` + `K` doesn't work...